### ISSUE TITLE
Installer blue screens my system with system_service_exception.

### ISSUE THREAD

**Issue by cedral** [nmap/npcap#253]
Windows 10 Pro 1909
it worked a couple of times then I installed the Microsoft KM-TEST Loopback adapter. After doing that npcap installation blue screens every time. Removing the adapter did not fix it. 
I have a minidump from the version in the current wireshark installer if that will help. I also tried the 1.0 installer from your website and it did the same thing. 
Help: I need to sniff loopback traffic to figure out why I am getting a tcp reset in the middle of a read when I didn't send any additional data on the closed connection. Rawshark doesn't work. The packets aren't in order. The syn ack shows up before the syn and I the acks are showing up before the data so the acks are marked as for unseen data and the data is being shown as a spurious resend. I need to use an actual driver in hopes that it will be able to sniff in order.

**Comment by dmiller-nmap** [comment]
Thanks for this report. The minidump would be very helpful, and you can send it privately to dmiller@nmap.com.

If the Npcap 1.00 installer did not succeed, it is most likely a crash in the existing installation. To recover, disable the npcap driver service (as Administrator run `sc.exe config npcap start= disabled`) and reboot, then try the new installation or uninstallation again.

**Comment by paulcsiki** [comment]
I've sent you an email with the minidumps.

**Comment by dmiller-nmap** [comment]
@paulcsiki We determined via email that your issue is #268. Do you have reason to believe this is the same issue as @cedral originally reported?

**Comment by paulcsiki** [comment]
@dmiller-nmap, no, I just thought it was related as it had the same stop code.
